### ISSUE TITLE
error running games (OSX, latest dev code)

### ISSUE THREAD

**Issue by lordmauve** [lordmauve/pgzero#64]
Originally reported by: **darcy_clark (Bitbucket: [darcy_clark](https://bitbucket.org/darcy_clark), GitHub: Unknown)**

----------------------------------------

I checked out the latest code - installed it via 'pip3 install .'

wrote a simple test game:
```
  WIDTH = 500
  HEIGHT = 500

  RED = 200, 0, 0
  BOX = Rect((20, 20), (100, 100))

  def draw():
      screen.fill((0, 0, 0))
      screen.draw.rect(BOX, RED)

  def update():
      BOX.left += 2
      if BOX.left > WIDTH:
          BOX.right = 0
```
then tried to run it via :

$ pgzrun pgzero_test.py 

Traceback (most recent call last):
  File "/Users/darcy/.pyenv/versions/3.6.3/bin/pgzrun", line 11, in <module>
    load_entry_point('pgzero==1.2a1', 'console_scripts', 'pgzrun')()
  File "/Users/darcy/.pyenv/versions/3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 572, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/Users/darcy/.pyenv/versions/3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2755, in load_entry_point
    return ep.load()
  File "/Users/darcy/.pyenv/versions/3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2408, in load
    return self.resolve()
  File "/Users/darcy/.pyenv/versions/3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2414, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Users/darcy/.pyenv/versions/3.6.3/lib/python3.6/site-packages/pgzero/runner.py", line 14, in <module>
    from . import builtins
  File "/Users/darcy/.pyenv/versions/3.6.3/lib/python3.6/site-packages/pgzero/builtins.py", line 4, in <module>
    from . import tone
  File "/Users/darcy/.pyenv/versions/3.6.3/lib/python3.6/site-packages/pgzero/tone.py", line 62, in <module>
    player_thread = Thread(target=_play_thread)
NameError: name '_play_thread' is not defined

The game works with the older released version.

----------------------------------------
- Bitbucket: https://bitbucket.org/lordmauve/pgzero/issue/64

**Comment by lordmauve** [comment]
*Original comment by* **Daniel Pope (Bitbucket: [lordmauve](https://bitbucket.org/lordmauve), GitHub: [lordmauve](https://github.com/lordmauve))**:

----------------------------------------

Fix bad re-ordering of code (fix #64)

**Comment by lordmauve** [comment]
*Original comment by* **Daniel Pope (Bitbucket: [lordmauve](https://bitbucket.org/lordmauve), GitHub: [lordmauve](https://github.com/lordmauve))**:

----------------------------------------

Fixed. Thanks for catching this.
